routes/definitions.js

```diff
@@ -17,7 +17,8 @@ async function getDefinition(request, response) {
   const coordinates = utils.toEntityCoordinatesFromRequest(request)
   const pr = request.params.pr
   // if running on localhost, allow a force arg for testing without webhooks to invalidate the caches
-  const force = request.hostname.includes('localhost') ? request.query.force || false : false
+
+  const force = request.query.force ? true : false
```

Comment:
Not sure what the ternary is doing here... This reads to me like `if request.query.force == true: return true, else false`. 

If `request.query.force` returns a boolean then you can drop the ternary, but if it's a string then you'll have to evaluate it like `request.query.force === 'true' ? true : false`

Reply:
Good point, revising.